src/features/feed/components/NearestFreshMartSection.jsx

```diff
@@ -0,0 +1,197 @@
+                <BukkasToExploreSection
+                  push={push}
+                  promotedBukkas={fetchedPromotedBukkas}
+                  fetchedCuisines={fetchedCuisines}
+                />
+                }
+                <Container
+                  classNames={mapContainerDisplay(displayMap)}
+                >
+                  <div className={mapContentDisplay(displayMap)}>
+                    <NearestFreshOrMart
+                      delivery
+                      classNames={displayMap ? 'col-12' : 'col-xl-4 col-md-6 col-sm-12'}
+                      title={displayMap ? '' : 'Nearby'}
+                      bukkaData={[...nearbyBukkas]}
+                      imageHeight={displayMap ? 'map-img-height' : 'img-height'}
+                      currentPage={currentPage}
+                      errorMessage={errorMessage}
+                      push={push}
+                    />
+                  </div>
+                  <div className={mapDisplay(displayMap)}>
+                    <Map restaurants={nearbyBukkas} coordinates={coordinates} />
+                  </div>
+                  {/* display carousel for map on small & medium screen */}
+                  {displayMap && (
+                    <div
+                      className="d-flex d-md-flex d-lg-none d-xl-none px-0 col-12"
+                    >
+                      <Carousel
+                        delivery
+                        noOfImagesShown={1}
+                        xl={1}
+                        lg={1}
+                        md={1}
+                        sm={1}
+                        slideItems={[...favorites, ...freeDelivery]}
+                        imageHeight="img-fluid"
+                        classNames="col-lg-6 col-md-6 col-sm-12 col-12"
+                      />
+                    </div>
+                  )}
+                </Container>
+              </div>
+            </div>
+          </ExploreSection>
+        </div>
+      )}
+    </div>
+  );
+};
+
+const mapStateToProps = ({
+  loadingReducer: { status: loading },
+  deliveryModeReducer: { mode },
+  bukkasReducer: { fetchedBukkas, status, currentPage, errorMessage },
+  selectedLocationReducer: { coordinates },
+  promotionReducer: { fetchedBukkas: fetchedPromotedBukkas },
+  cuisineReducer: { fetchedBukkas: fetchedCuisines }
+}) => ({
+  fetchedBukkas,
+  fetchedCuisines,
+  fetchedPromotedBukkas,
+  status,
+  currentPage,
+  coordinates,
+  mode,
+  errorMessage,
+  loading,
+});
+
+export default connect(
+  mapStateToProps,
+  { fetchNearbyBukkas: fetchBukkas },
+)(NearestFreshMartSection);
+
+NearestFreshMartSection.propTypes = {
+  mode: PropTypes.string.isRequired,
+  push: PropTypes.func.isRequired,
+  coordinates: PropTypes.arrayOf(PropTypes.number).isRequired,
+  fetchedBukkas: PropTypes.shape({
+    nearbyBukkas: PropTypes.arrayOf(PropTypes.shape({})),
+  }).isRequired,
+  fetchNearbyBukkas: PropTypes.func.isRequired,
+};
```

Comment:
You don't need a new component. Use `OtherComponent.jsx` for both fresh and mart. Remember to adjust it respectively in the index.js file

Reply:
New NearestFreshMartSection.js component deleted